File: MODUL 4/PRAK402-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
def urutkan_bilangan(batas):
    # Bilangan ganjil
    for i in range(1, batas + 1, 2):
        print(i, end=" ")
    print()

    # Bilangan genap terbalik
    for i in range(batas if batas % 2 == 0 else batas - 1, 1, -2):
        print(i, end=" ")
    print()

if __name__ == "__main__":
    # Membaca input dari pengguna
    print("Input")
    batas = int(input())

    # Menampilkan output
    print("\nOutput")
    urutkan_bilangan(batas)

File: MODUL 4/PRAK402-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

void urutkan_bilangan(int batas) {
for (int i = 1; i <= batas; i += 2) {
    printf("%d ", i);
    }
    printf("\n");

for (int i = (batas % 2 == 0) ? batas : batas - 1; i >= 2; i -= 2) {
        printf("%d ", i);
    }
    printf("\n");
}

int main() {
int batas;

    printf("Input\n");
    scanf("%d", &batas);

    printf("\nOutput\n");
    urutkan_bilangan(batas);

return 0;
}
